controllers/topics.controller.js

```diff
@@ -0,0 +1,7 @@
+const { fetchAllTopics } = require ("../models/topics.model")
+
+exports.getAllTopics = (req, res, next) => { 
+    fetchAllTopics() 
+    .then((body) => res.status(200).send({body}))
```

Comment:
We do want to send an object back, but we'd prefer a descriptive key here - e.g topics as body is a bit generic and doesn't tell us what's been sent back